src/components/DefinitionEntry.js

```diff
@@ -280,17 +280,17 @@ export default class DefinitionEntry extends React.Component {
 
     // TODO: find a way of calling this method less frequently. It's relatively expensive.
     const definition = this.foldFacets(rawDefinition, this.props.activeFacets)
-    const { licensed, described } = definition
-    const initialFacets =
-      get(described, 'facets') || this.isSourceComponent(definition.coordinates)
-        ? ['Core', 'Data', 'Dev', 'Doc', 'Examples', 'Tests']
-        : ['Core']
+    const { licensed } = definition
+    // const initialFacets =
```

Comment:
Is this update part of the addressed issue?

Reply:
This fixes/clear an eslint warning. `no-unused-vars`